modules/guAnalyticsAdapter.js

```diff
@@ -67,6 +75,17 @@ function sendAll() {
   }
 }
 
+function trackBidWon(args){
+    const event = createHbEvent( args.bidderCode, 'bidwon',
+                                 args.adUnitCode,
+                                 args.auctionId,
+                                 undefined, // timeToRespond
+                                 undefined, // startTime
+                                 args.bidId
+                               );
```

Comment:
I think we only need 'bidWon', auctionId and bidId.

Reply:
Yep. I will remove all the other stuff :)